sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulseCoordinator.java

```diff
@@ -58,13 +58,12 @@ public void run() {
             }
         } catch (InterruptedException e) {
             Thread.currentThread().interrupt();
-        } catch (Throwable t) {
-            if (t instanceof Error) {
-                throw (Error) t;
-            }
+            logger.error("QuickPulseCoordinator was interrupted", e);
+        } catch (Exception exception) {
+            // Not supposed to happen
+            logger.error("QuickPulseCoordinator failed", exception);
```

Comment:
(this is my preference to be safe, even though prior code was different)
```suggestion
        } catch (Throwable t) {
            // Not supposed to happen
            logger.error("QuickPulseCoordinator failed", t);
```